addons/bus/models/res_users.py

```diff
@@ -1,25 +0,0 @@
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-
-class ResUsers(models.Model):
-    _name = 'res.users'
-    _inherit = ["res.users", "bus.listener.mixin"]
```

Comment:
I would keep the mixin here, there is no reason to artificially limit it to mail module.
This change is unrelated to presence.